acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -7,6 +7,10 @@
 PXP_LOG_FILE_CYGPATH = '/cygdrive/c/ProgramData/PuppetLabs/pxp-agent/var/log/pxp-agent.log'
 PXP_LOG_FILE_POSIX = '/var/log/puppetlabs/pxp-agent/pxp-agent.log'
 
+PXP_EXE_DIR_CYGPATH = "/cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/pxp-agent/bin"
+PXP_EXE_DIR_CYGPATH_x86 = "/cygdrive/c/Program\\ Files\\ \\(x86\\)/Puppet\\ Labs/Puppet/pxp-agent/bin"
+PXP_EXE_DIR_POSIX = "/opt/puppetlabs/puppet/bin"
+
```

Comment:
Premature declaration of a constant, they're here so pxp_agent_dir has something to return, and only so pxp_agent_dir has something to return.  They should just be string literals in the pxp_agent_dir function.